source/devices/AM65X/linux/_Release_Specific_RT_Kernel_Performance_Guide.rst

```diff
@@ -373,105 +384,211 @@ Running the following commands will trigger netperf clients to measure UDP burst
 
-CPSW/CPSW2g/CPSW3g Ethernet Driver 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-- CPSW2g: AM65x, J7200, J721e
-- CPSW3g: AM64x
+ICSSG Ethernet Driver 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. rubric::  TCP Bidirectional Throughput Interrupt Pacing
-   :name: CPSW2g-tcp-bidirectional-throughput-interrupt-pacing
+.. rubric::  TCP Bidirectional Throughput 
+   :name: tcp-bidirectional-throughput
 
-.. csv-table::
+.. csv-table:: ICSSG TCP Bidirectional Throughput
     :header: "Command Used","am654x-evm: THROUGHPUT (Mbits/sec)","am654x-evm: CPU Load % (LOCAL_CPU_UTIL)"
 
-    "netperf -H 192.168.0.1 -j -c -C -l 60 -t TCP_STREAM; netperf -H 192.168.0.1 -j -c -C -l 60 -t TCP_MAERTS","1385.92","33.09"
+    "netperf -H 192.168.2.1 -j -c -C -l 60 -t TCP_STREAM; netperf -H 192.168.2.1 -j -c -C -l 60 -t TCP_MAERTS","1021.47","45.64"
 
-.. rubric::  UDP Throughput 
-   :name: CPSW2g-udp-throughput-0-loss
+.. csv-table:: ICSSG UDP Ingress Throughput 0 loss
+    :header: "Frame Size(bytes)","am654x-evm: UDP Datagram Size(bytes) (LOCAL_SEND_SIZE)","am654x-evm: THROUGHPUT (Mbits/sec)","am654x-evm: Packets Per Second (kPPS)","am654x-evm: CPU Load %"
 
-Table:  **CPSW2g UDP Egress Throughput 0 loss**
+    "256","210.00","21.34","13.00","7.80"
 
-.. csv-table::
-    :header: "Frame Size(bytes)","am654x-evm: UDP Datagram Size(bytes) (LOCAL_SEND_SIZE)","am654x-evm: THROUGHPUT (Mbits/sec)","am654x-evm: Packets Per Second (kPPS)","am654x-evm: CPU Load % (LOCAL_CPU_UTIL)"
+|
 
-    "64","18.00","10.59","74.00","46.60"
-    "128","82.00","47.45","72.00","46.85"
-    "256","210.00","120.51","72.00","46.18"
-    "1024","978.00","612.40","78.00","44.89"
-    "1518","1472.00","788.20","67.00","47.67"
+PCIe Driver
+-------------------------
 
-Table:  **CPSW2g UDP Ingress Throughput 0 loss**
+PCIe-ETH
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. csv-table::
-    :header: "Frame Size(bytes)","am654x-evm: UDP Datagram Size(bytes) (LOCAL_SEND_SIZE)","am654x-evm: THROUGHPUT (Mbits/sec)","am654x-evm: Packets Per Second (kPPS)","am654x-evm: CPU Load % (LOCAL_CPU_UTIL)"
+.. csv-table:: PCIe Ethernet performance
+    :header: "TCP Window Size(Kbytes)","am654x-evm: Bandwidth (Mbits/sec)"
 
-    "64","18.00","3.61","25.00","12.00"
-    "128","82.00","16.60","25.00","12.50"
-    "256","210.00","44.01","26.00","13.19"
-    "1518","1472.00","317.95","27.00","14.15"
+    "8","0.00"
+    "16","0.00"
+    "32","0.00"
+    "64","0.00"
+    "128","0.00"
+    "256","0.00"
 
-Table:  **CPSW2g UDP Ingress Throughput possible loss**
+EMMC Driver
+-----------
+.. warning::
 
-.. csv-table::
-    :header: "Frame Size(bytes)","am654x-evm: UDP Datagram Size(bytes) (LOCAL_SEND_SIZE)","am654x-evm: THROUGHPUT (Mbits/sec)","am654x-evm: Packets Per Second (kPPS)","am654x-evm: CPU Load % (LOCAL_CPU_UTIL)","am654x-evm: Packet Loss %"
+  **IMPORTANT**: The performance numbers can be severely affected if the media is
```

Comment:
Drop IMPORTANT since https://github.com/TexasInstruments/processor-sdk-doc/pull/87/files#r1878698894 handles it